dsgrid_project/dimensions/subset/sectors.csv

```diff
@@ -3,3 +3,4 @@ com,x,,,
 ind,,x,,
 res,,,x,
 trans,,,,x
+unspecified,,,,
```

Comment:
If you don’t think this should be required, please raise the concern.

Reply:
I think this is fine/in the spirit of making people over-specify/over-match things.